cmd/node/config/config.toml

```diff
@@ -766,3 +766,6 @@
 
 [Logs]
     LogFileLifeSpanInSec = 86400
+
+[TotalStakedValueAPI]
+    TotalStakedValueAPICacheDurationInMinutes = 100
```

Comment:
No, the node should try not to cache values. That is the job of the middle layer that will query the node.